src/utils/ensureExtension.ts

```diff
@@ -22,7 +24,5 @@ export namespace EnsureExtension {
             ];
             throw new Error(options.noExtensionErrorMessage);
```

Comment:
the error message may be updated that user have a lower version of extension?

Reply:
This name is confusing. It's not that there are no plugins, but rather that there are no suitable ones.